scripts/package-harvester-os

```diff
@@ -22,7 +22,6 @@ LOGGING_VERSION=${RANCHER_LOGGING_CHART_VERSION}
 echo "Rancher monitoring version: ${MONITORING_VERSION}"
 echo "Rancher logging version: ${LOGGING_VERSION}"
 echo "Harvester eventrouter image tag: ${HARVESTER_EVENTROUTER_FULL_TAG}"
-rm -rf ${addons_path}
```

Comment:
BTW: use env `LOCAL_ADDONS_SRC` to determin if rm or not ? thanks.

Reply:
I don't think `LOCAL_ADDONS_SRC` is actually passed through to the build environment right now. Why do this? Is it important that the addons source be removed from the build environment at this point?

Reply:
no speicial purpose, just avoid remaining file affecting others;  skip `rm` is ok

Reply:
cool, thanks!